Kubernetes/windows/helper.psm1

```diff
@@ -44,20 +33,19 @@ function CleanupOldNetwork($NetworkMode)
         Write-Host ($hnsNetwork | ConvertTo-Json -Depth 10) 
         Remove-HnsNetwork $hnsNetwork
     }
-    # Wait for the interface to come back with IP
-    Start-Sleep 10
 }
 
-function WaitForNetwork($NetworkMode)
+function WaitForNetwork($NetworkName)
 {
     # Wait till the network is available
-    while( !(Get-HnsNetwork -Verbose | ? Type -EQ $NetworkMode.ToLower()) )
+    while( !(Get-HnsNetwork -Verbose | ? Name -EQ $NetworkName.ToLower()) )
     {
         Write-Host "Waiting for the Network to be created"
-        Start-Sleep 10
+        Start-Sleep 1
     }
 }
 
+
```

Comment:
Cgroups per qos should be set explicitly to false as it does not apply for Windows?
We should append to arglist of kubelet
 "--cgroups-per-qos=false", "--enforce-node-allocatable="""""

Reply:
I fixed this, but we still need to refactor the scripts to remove duplicate functions and make it more configurable.